index.html

```diff
@@ -95,12 +96,12 @@ <h2>Use the <strong>Service Mesh Performance</strong> standard</h2>
 				<h2>What are <strong>configuration best practices</strong> and patterns?</h2>
 				<p> Assess your service mesh configuration against deployment and
 					operational best practices with Meshery's configuration validator.
-				<div align="center">
-					See the <a href="https://docs.meshery.io">service mesh patterns</a>
+				<div align="center" class="button adopter-button">
+					<a href="https://docs.meshery.io">See the service mesh patterns</a>
```

Comment:
```suggestion
					<a href="https://docs.meshery.io/functionality/pattern-management">See the service mesh patterns</a>
```
Let's change the link to this.